File: Praktikum6/Bahasa C/PRAK602-2010817210023-MuhammadAuliaAkbar.c
#include <stdio.h>

int main(){
    int nInput;
    scanf("%d", &nInput);
    int inputke[nInput];


    for(int i = 1; i <= nInput; i++){
        scanf("%d", &inputke[i-1]);
    }

    for(int i = 1; i <= nInput; i++){
        printf("%d ", inputke[i-1] * i);
    }

}

File: Praktikum6/Bahasa Python/PRAK602-2010817210023-MuhammadAuliaAkbar.py
nInput = int(input())
inputke = []

# Input elemen
input_str = input()
input_list = input_str.split()

for i in range(nInput):
    inputke.append(int(input_list[i]))

# Output hasil perhitungan
for i in range(1, nInput + 1):
    print(inputke[i - 1] * i, end=" ")
